### PR TITLE
Update Nations.json

### PR DESCRIPTION
**Pull request by Thienanbad**

### CODE REVIEW

File: jsons/Nations.json [hunk 1]
```diff
@@ -2219,11 +2219,11 @@
...
 		"hateHello": "What do you want?",
 		"tradeRequest": "Better hurry, I may change my mind.",
-		"outerColor": [0,204,126],
-		"innerColor": [128,3,28],
+		"outerColor": [218,37,29],
+		"innerColor": [255,255,0],
 		"uniqueName": "Tam Giao",
 		"uniques": ["[+2 Food, +1 Production] from every [Marsh]","[+1 Faith] from every [Jungle]"],
-		"cities": ["Ha Noi","Thanh Pho Ho Chi Minh","Hai Phong","Thanh Pho Hue","Can Tho","Da Nang","Bien Hoa","Phan Rang","Quang Ngai","Quang Ninh","Tuyen Quang","Ha Tinh","Tam Ky"]
+		"cities": ["Ha Noi","Tp.Ho Chi Minh","Hai Phong","Thanh Pho Hue","Can Tho","Da Nang","Bien Hoa","Phan Rang","Quang Ngai","Quang Ninh","Tuyen Quang","Ha Tinh","Tam Ky"]
```
Comment (yairm210) [inline jsons/Nations.json:2226]: What is this about?

File: jsons/Nations.json [hunk 2]
```diff
@@ -2219,11 +2219,11 @@
 		"neutralHello": "Greetings.",
 		"hateHello": "What do you want?",
 		"tradeRequest": "Better hurry, I may change my mind.",
-		"outerColor": [0,204,126],
-		"innerColor": [128,3,28],
+		"outerColor": [218,37,29],
+		"innerColor": [255,255,0],
```
Comment (yairm210) [inline jsons/Nations.json:2223]: Where are these colors from? Base game?

File: jsons/Nations.json (hunk 1 above)
Comment (Thienanbad) [reply jsons/Nations.json:2226]: > What is this about?
Bcs there arr some province name so i have to change it back to city name

File: jsons/Nations.json (hunk 2 above)
Comment (Thienanbad) [reply jsons/Nations.json:2223]: Color from Vietnam flag

**Comment by yairm210** [comment]
You need to show a picture of how it looks,  like a unit icon. I don't know if it conflicts with other nations colors...

**Comment by Thienanbad** [comment]
It is in nationicon

Vào 16:32, Th 3, 13 thg 9, 2022 Yair Morgenstern ***@***.***>
đã viết:

**Comment by Thienanbad** [comment]
https://github.com/ravignir/RekMOD/blob/7db7cf21659d6d4f78c7c9420d33f94bd5194c19/Images/NationIcons/Vietnam.png

Vào 16:32, Th 3, 13 thg 9, 2022 Yair Morgenstern ***@***.***>
đã viết:

**Pull request closed without merging**